docs/install/reference.md

```diff
@@ -94,6 +94,8 @@ is given, milliseconds will be assumed. If set to `0`, ArgoCD Image Updater
 will exit after the first run, effectively disabling the interval. Default
 value is `2m0s`.
 
+Can also be set using the *IMAGE_UPDATER_INTERVAL* environment variable.
+
```

Comment:
I think we should descriibe the order the precedence here: if the param is not set, then check the env var, and if neither the param or env var is set, then fall back to the default value 2m.